src/axom/inlet/inlet_utils.cpp

```diff
@@ -103,7 +104,14 @@ bool verifyRequired(const axom::sidre::Group& target,
         "specified: {1}",
         type,
         target.getPathName());
-      SLIC_WARNING(msg);
+      if(errors)
+      {
+        errors->push_back({Path {target.getPathName()}, msg});

```

Comment:
Is `Path` the same as `axom::Path`?
Suggestion: It might be clearer to readers of the code to include the `axom` namespace.

Reply:
Yep, this is just the Path class